examples/bri-3/src/bri/workgroup/worksteps/agents/worksteps.agent.ts

```diff
@@ -0,0 +1,52 @@
+import { BadRequestException, Injectable, NotFoundException } from '@nestjs/common';
+import { Workstep } from '../models/workstep';
+
+import { v4 as uuidv4 } from 'uuid';
+import { NAME_EMPTY_ERR_MESSAGE, NOT_FOUND_ERR_MESSAGE } from "../api/err.messages";
+import { WorkstepStorageAgent } from '../persistence/workstepsStorage.agent';
+
+@Injectable()
+export class WorkstepAgent {
+  constructor(private storageAgent: WorkstepStorageAgent) {}
+  public throwIfCreateWorkstepInputInvalid(name: string, version: string, status: string, workgroupId: string, securityPolicy: string, privacyPolicy: string) {
+    // This is just an example, these fields will be validated on the DTO validation layer
```

Comment:
why we need to add example here too? we have it on one place, im fine with it, but no need for so much code that we will clean up anyways i think

Reply:
Removed this example from the worksteps agent and its associated usages and test